eng/testing/RunnerTemplate.cmd

```diff
@@ -62,7 +62,13 @@ pushd %EXECUTION_DIR%
 @echo off
 popd
 echo ----- end %DATE% %TIME% ----- exit code %ERRORLEVEL% ----------------------------------------------------------
-exit /b %ERRORLEVEL%
+:: The helix work item should not exit with non-zero if tests ran and produced results
+:: The special console runner for runtime returns 1 when tests fail
+if %ERRORLEVEL%==1 (
+  exit /b 0
+) else (
+  exit /b %ERRORLEVEL%
```

Comment:
Same here

```suggestion
  exit /b 1
```

Reply:
This if is correct, if %ERRORLEVEL% is 1, return 0, otherwise return %ERRORLEVEL%